tgui/packages/tgui/interfaces/TrackedPlaytime.js

```diff
@@ -71,7 +72,11 @@ export const TrackedPlaytime = (props, context) => {
           <Box>
             <Section title="Total">
               <PlaytimeSection
-                playtimes={{
+                playtimes={adminTime ? {
+                  "Ghost": ghostTime,
+                  "Living": livingTime,
+                  "Admin": adminTime,
```

Comment:
damn im good

Reply:
Smelly, but it's whatever.

Reply:
Is it less smelly now?

Reply:
i think he was talking about my code, i don't think you should be allowing nulls to come up in PlaytimeSection tbh

Reply:
It's more a case of doing filtering JS-side instead of DM-side. There's a few values that are always passed to the interface even when they're 0 (I tested with a 0h account locally). I can certainly make sure that only non-zero values are sent JS-side.

Reply:
![image](https://user-images.githubusercontent.com/24975989/113968174-7c83e880-982a-11eb-8082-9fa77e7de0f6.png)

Figured it would be nice to clean it up with less information.